<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Bartosz Baranowski - Bioinformatician</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet">
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
<div id="header-placeholder"></div>
<main class="container mt-4">

    <div class="row">
        <div class="col-md-6">
            <div class="card mb-4">
                <img src="https://blogs.biomedcentral.com/bmcblog/files/2009/06/bmc%20research%20notes%20logo%20small.jpg"
                     class="card-img-top custom-icon-bmc" alt="magazine icon">
                <div class="card-body">
                    <h3 class="card-title">KINtaro: protein kinase-like database</h3>
                    <p class="card-text"><b>Authors:</b> B. Baranowski, M. Krysińska, M.Gradowski</p>
                    <p class="card-text"><b>Published in:</b> BMC Research Notes, 16 February 2024 </p>
                    <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#abstract1" role="button"
                           aria-expanded="false" aria-controls="abstract1">
                            Show/Hide Abstract
                        </a>
                    </p>
                    <div class="collapse" id="abstract1">
                        <div class="card card-body">
                            <b>Abstract:</b> The superfamily of protein kinases features a common Protein Kinase-like
                            (PKL) three-dimensional fold. Proteins with PKL structure can also possess enzymatic
                            activities other than protein phosphorylation, such as AMPylation or glutamylation. PKL
                            proteins play a vital role in the world of living organisms, contributing to the survival of
                            pathogenic bacteria inside host cells, as well as being involved in carcinogenesis and
                            neurological diseases in humans. The superfamily of PKL proteins is constantly growing.
                            Therefore, it is crucial to gather new information about PKL families. To this end, the
                            KINtaro database (http://bioinfo.sggw.edu.pl/kintaro/) has been created as a resource for
                            collecting and sharing such information. KINtaro combines protein sequence information and
                            additional annotations for more than 70 PKL families, including 32 families not associated
                            with PKL superfamily in established protein domain databases. KINtaro is searchable by
                            keywords and by protein sequence and provides family descriptions, sequences, sequence
                            alignments, HMM models, 3D structure models, experimental structures with PKL domain
                            annotations and sequence logos with catalytic residue annotations.
                        </div>
                    </div>
                    <a href="https://link.springer.com/article/10.1186/s13104-024-06713-y"
                       class="btn btn-secondary mt-2">Read More</a>
                </div>
            </div>
        </div>

        <div class="col-md-6">
            <div class="card mb-4">
                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRDd0QpE9l8IZYpT6_AAWAE420E4IMjgwBhJ1RvO8cvSw&s"
                     class="card-img-top custom-icon-peerj" alt="magazine icon">
                <div class="card-body">
                    <h3 class="card-title">Protein family neighborhood analyzer—ProFaNA</h3>
                    <p class="card-text"><b>Authors:</b> B. Baranowski, K.Pawłowski</p>
                    <p class="card-text"><b>Published in:</b> PeerJ, July 21, 2023 </p>
                    <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#abstract2" role="button"
                           aria-expanded="false" aria-controls="abstract2">
                            Show/Hide Abstract
                        </a>
                    </p>
                    <div class="collapse" id="abstract2">
                        <div class="card card-body">
                            <b>Abstract:</b> Functionally related genes are well known to be often grouped in close
                            vicinity in the genomes, particularly in prokaryotes. Notwithstanding the diverse
                            evolutionary mechanisms leading to this phenomenon, it can be used to predict functions of
                            uncharacterized genes. Here, we provide a simple but robust statistical approach that
                            leverages the vast amounts of genomic data available today. Considering a protein domain as
                            a functional unit, one can explore other functional units (domains) that significantly often
                            occur within the genomic neighborhoods of the queried domain. This analysis can be performed
                            across different taxonomic levels. Provisions can also be made to correct for the uneven
                            sampling of the taxonomic space by genomic sequencing projects that often focus on large
                            numbers of very closely related strains, e.g., pathogenic ones. To this end, an optional
                            procedure for averaging occurrences within subtaxa is available. Several examples show this
                            approach can provide useful functional predictions for uncharacterized gene families, and
                            how to combine this information with other approaches. The method is made available as a web
                            server at http://bioinfo.sggw.edu.pl/neighborhood_analysis.

                        </div>
                    </div>
                    <a href="https://peerj.com/articles/15715/"
                       class="btn btn-secondary mt-2">Read More</a>
                </div>
            </div>
        </div>
        <div class="col-md-6">
            <div class="card mb-4">
                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQvpFyyFVxxfaAtdPePbp4FQGeu7EkIhnBnla9wi_M44A&s"
                     class="card-img-top custom-icon-peerj" alt="magazine icon">
                <div class="card-body">
                    <h3 class="card-title">Pan-kinome of <i>Legionella</i> expanded by a bioinformatics survey</h3>
                    <p class="card-text"><b>Authors:</b> M. Krysińska, B. Baranowski, B. Deszcz, K. Pawłowski, M.
                        Gradowski</p>
                    <p class="card-text"><b>Published in:</b> Nature - Scientific Reports, 16 December 2022 </p>
                    <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#abstract3" role="button"
                           aria-expanded="false" aria-controls="abstract3">
                            Show/Hide Abstract
                        </a>
                    </p>
                    <div class="collapse" id="abstract3">
                        <div class="card card-body">
                            <b>Abstract:</b> The pathogenic Legionella bacteria are notorious for delivering numerous
                            effector proteins into the host cell with the aim of disturbing and hijacking cellular
                            processes for their benefit. Despite intensive studies, many effectors remain
                            uncharacterized. Motivated by the richness of Legionella effector repertoires and their
                            oftentimes atypical biochemistry, also by several known atypical Legionella effector kinases
                            and pseudokinases discovered recently, we undertook an in silico survey and exploration of
                            the pan-kinome of the Legionella genus, i.e., the union of the kinomes of individual
                            species. In this study, we discovered 13 novel (pseudo)kinase families (all are potential
                            effectors) with the use of non-standard bioinformatic approaches. Together with 16 known
                            families, we present a catalog of effector and non-effector protein kinase-like families
                            within Legionella, available at http://bioinfo.sggw.edu.pl/kintaro/. We analyze and discuss
                            the likely functional roles of the novel predicted kinases. Notably, some of the kinase
                            families are also present in other bacterial taxa, including other pathogens, often
                            phylogenetically very distant from Legionella. This work highlights Nature’s ingeniousness
                            in the pathogen–host arms race and offers a useful resource for the study of infection
                            mechanisms.

                        </div>
                    </div>
                    <a href="https://www.nature.com/articles/s41598-022-26109-x"
                       class="btn btn-secondary mt-2">Read More</a>
                </div>
            </div>
        </div>
        <div class="col-md-6">
            <div class="card mb-4">
                <img src="https://www.biochemistry.org/assets/colour-logo.png"
                     class="card-img-top custom-icon-bsm" alt="magazine icon">
                <div class="card-body">
                    <h3 class="card-title">The expanding world of protein kinase-like families in bacteria: forty
                        families and counting</h3>
                    <p class="card-text"><b>Authors:</b> M.Gradowski, B.Baranowski, K.Pawłowski</p>
                    <p class="card-text"><b>Published in:</b> Biochemical Society Transactions, 17 July 2020 </p>
                    <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#abstract4" role="button"
                           aria-expanded="false" aria-controls="abstract3">
                            Show/Hide Abstract
                        </a>
                    </p>
                    <div class="collapse" id="abstract4">
                        <div class="card card-body">
                            <b>Abstract:</b> The protein kinase-like clan/superfamily is a large group of regulatory,
                            signaling and biosynthetic enzymes that were historically regarded as typically eukaryotic
                            proteins, although bacterial members have also been known for a long time. In this review,
                            we explore the diversity of bacterial protein kinase like families, and discuss functional
                            versatility of these enzymes, both the ones acting within the bacterial cell, and those
                            acting within eukaryotic cells as effectors during infection. We focus on novel bacterial
                            kinase-like families discovered in the last five years. A bioinformatics perspective is held
                            here, hence sequence and structure comparison overview is presented, and also a comparison
                            of genomic neighbourhoods of the families. We perform a phylum-level census of the families.
                            Also, we discuss apparent pseudokinases that turned out to perform alternative catalytic
                            functions by repurposing their atypical kinase-like active sites. We also highlight some
                            ‘unpopular' kinase-like families that await characterisation.

                        </div>
                    </div>
                    <a href="https://portlandpress.com/biochemsoctrans/article-abstract/48/4/1337/225822/The-expanding-world-of-protein-kinase-like"
                       class="btn btn-secondary mt-2">Read More</a>
                </div>
            </div>
        </div>
        <div class="col-md-6">
            <div class="card mb-4">
                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTCdnjjc8y4Z_4IOvOokTeeqijDD8FNSB9BKsC63cen1A&s"
                     class="card-img-top custom-icon-3bt" alt="magazine icon">
                <div class="card-body">
                    <h3 class="card-title">Purification, characterization and cloning of a chitinase from
                        Stenotrophomonas rhizophila G22
                    </h3>
                    <p class="card-text"><b>Authors:</b> U. Jankiewicz, B Baranowski, M. Swiontek-Brzezinska, M Frąk</p>
                    <p class="card-text"><b>Published in:</b> 3BioTech, 10 December 2019 </p>
                    <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#abstract5" role="button"
                           aria-expanded="false" aria-controls="abstract3">
                            Show/Hide Abstract
                        </a>
                    </p>
                    <div class="collapse" id="abstract5">
                        <div class="card card-body">
                            <b>Abstract:</b> In the presented research the extracellular chitinase of Stenotrophomonas
                            rhizophila G22 was biochemically and molecularly characterized. The studied enzyme was
                            purified from a 72-h bacterial culture about 14 times, with a recovery of 63%. The molecular
                            weight of the purified protein was estimated at 50 kDa by SDS-PAGE. The enzyme showed high
                            activity against colloidal chitin. Significantly lower activities were observed with native
                            chitin powder and chitosan. Adsorption of the enzyme to colloidal chitin and to powdered
                            chitin at the level of 75% and 37%, respectively, was observed after 30 min of reaction.
                            Optimum temperature and pH were 37 °C and 5.9, respectively. The enzyme demonstrated higher
                            activity against nitrophenyl-β D N, N′, N″-triacetylchitotriose and approx. 5 times lower
                            activity for 4-nitrophenyl-N, N′-diacetyl-β-D-chitobiose. The enzyme is an endochitinase,
                            which is confirmed by the Km and Vmax values determined in the studies. S. rhizophila G22
                            endochitinase was inhibited in the presence of cysteine-specific inhibitors, which indicates
                            the role of cysteine moieties in the mechanism of catalysis or in stabilisation of the
                            enzyme molecule. Also Ca2+ and Mn2+ ions may stabilise the protein's spatial structure. SDS
                            and ions: Fe2+, Cu2+, Co2+, Zn2+ inhibited the activity of enzyme. A full-length (2109 bp)
                            gene coding chitinase from S. rhizophila G22 was obtained. Four domains typical for
                            glycoside hydrolase family 18 (GH 18) chitinases were identified: catalytic Gly_18,
                            chitin-binding—ChtBD3, type-III fibronectin—FN3 and polycystic kidney disease domain—PKD
                            domain.

                        </div>
                    </div>
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6904715/"
                       class="btn btn-secondary mt-2">Read More</a>
                </div>
            </div>
        </div>

        <!-- Repeat the above structure for each publication -->
    </div>


</main>
<div id="footer-placeholder"></div>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"></script>
<script src="js/load.js"></script>
</body>
</html>
